src/sdk/staking.ts

```diff
@@ -159,7 +160,7 @@ async function metaStake(web3: Web3, address: string, isV2: boolean): Promise<St
  *  @param {string} account account details to fetch.
  * @returns {Promise<Stake | null>}
  */
-async function metaMyStake(web3: Web3, address: string, account: string, isV2: boolean): Promise<MyStake | null> {
+async function metaMyStake(web3: Web3, address: string, account: string, isV2: boolean, isV3?: boolean): Promise<MyStake | null> {
```

Comment:
just as in gov there's no need for those booleans, you have the address field overwrite